service/web/tabs/app/src/containers/BackfillStatusContainer.tsx

```diff
@@ -84,6 +93,10 @@ class BackfillStatusContainer extends React.Component<
     this.setState({ editing: {} })
   }
 
+  componentWillUnmount() {
```

Comment:
btw.. fixed this. Previously it kept polling status even after unloading the page. oops!

Reply:
nice!